UNRELEASED.md

```diff
@@ -12,6 +12,17 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
```

Comment:
Something bizarre going on here.

Reply:
Yea I'm not really sure how this happened 😂all of the previous changes were in my unrealeased.md file so when I added what I had changed it just pushed all of them.

Should I just delete all of the other ones in there besides my own since they're already merged into master?